app/web/src/newhotness/layout_components/AttributeInput.vue

```diff
@@ -17,10 +17,10 @@
           <TruncateWithTooltip>{{ displayName }}</TruncateWithTooltip>
           <div class="flex flex-row items-center ml-auto gap-2xs">
             <StatusIndicatorIcon
-              v-if="validation"
+              v-if="validation && validation.status !== 'Success'"
               v-tooltip="validation.message"
               type="qualification"
-              :status="validation.status === 'Success' ? 'success' : 'failure'"
```

Comment:
This will make the icon neutral, I believe. the `validations.status` values are uppercase, while the components expects lowercase values. 
The ideal thing would be setting `status="failure"` over here.